### PR TITLE
whisper-diff improvement: new until param

### PR DESCRIPTION
**Pull request by diuis**
Modified whisper.py and whisper-diff.py to accept a new parameter (--until) to improve the comparision of whisper files.

See [https://github.com/graphite-project/whisper/issues/182](url)

### CODE REVIEW

File: bin/whisper-diff.py [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
 import sys
+import time
 import optparse
 
 try:
   import whisper
 except ImportError:
   raise SystemExit('[ERROR] Please make sure whisper is installed properly')
 
+now = int( time.time() )
```
Comment (deniszh) [inline bin/whisper-diff.py]: You're declaring now variable here, but do not using it. Am I right?

**Comment by obfuscurity** [comment]
Unless the `.gitignore` changes are necessary let's remove them from this PR, please.

File: bin/whisper-diff.py (hunk 1 above)
Comment (diuis) [inline bin/whisper-diff.py]: Yes, you're right, my fault. I'll remove it.

**Comment by deniszh** [comment]
Now looks completely fine. If no objections - will merge this tomorrow.
Thanks for contribution, @diuis !

**Comment by diuis** [comment]
Hi @deniszh and @obfuscurity .
Is it possible to merge the pull request?
Thank you very much!

**Comment by deniszh** [comment]
Yes, sorry, was distracted. Thanks a lot, @diuis !

**Pull request merged**